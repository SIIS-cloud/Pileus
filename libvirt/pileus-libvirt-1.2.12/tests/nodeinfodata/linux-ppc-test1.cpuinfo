processor	: 0
cpu		: POWER7 (raw), altivec supported
clock		: 8.388608MHz
revision	: 2.3 (pvr 003f 0203)

processor	: 1
cpu		: POWER7 (raw), altivec supported
clock		: 8.388608MHz
revision	: 2.3 (pvr 003f 0203)
