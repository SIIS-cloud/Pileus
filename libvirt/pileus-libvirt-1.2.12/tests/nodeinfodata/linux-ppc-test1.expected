CPUs: 2/2, MHz: 8, Nodes: 1, Sockets: 1, Cores: 2, Threads: 1
